### PR TITLE
Correct exit codes

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by adiaaida** [comment]
test this please

File: dir.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Build Tools Version -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.17-prerelease</BuildToolsVersion>
+    <BuildToolsVersion>1.0.19-prerelease</BuildToolsVersion>
   </PropertyGroup>
```
Comment (weshaggard) [inline dir.props]: Is this version change intended?

File: src/dirs.proj [hunk 2]
```diff
@@ -28,7 +28,12 @@
   </PropertyGroup>
 
   <Import Project="$(ToolsDir)packages.targets" Condition="Exists('$(ToolsDir)packages.targets')" />
-  <Target Name="BuidDirPackages" AfterTargets="Build" DependsOnTargets="BuildPackages" />
+  <PropertyGroup Condition="Exists('$(ToolsDir)packages.targets')">
+    <TraversalBuildDependsOn>
+      $(TraversalBuildDependsOn);
+      BuildPackages;
+    </TraversalBuildDependsOn>
+  </PropertyGroup>
```
Comment (weshaggard) [inline src/dirs.proj]: I know this was my change but we should probably consider putting these in the individual targets files now.

**Comment by weshaggard** [comment]
The change looks good assuming there is a matching build tools package.

File: dir.props (hunk 1 above)
Comment (mmitche) [inline dir.props]: Yes.  Matt Cohn requested that I go to .19

File: src/dirs.proj (hunk 2 above)
Comment (mmitche) [inline src/dirs.proj]: I'll change that.

Comment (mmitche) [inline src/dirs.proj]: Actually, I'll merge this first.  Those targets live in buildtools, so I'll need to bump the version again and re-upload the package.  Will do as a separate change.

**Pull request merged**